src/dataframe/dataframe.jl

```diff
@@ -326,10 +327,11 @@ end
 # Will automatically add a new column if needed
 function insert_single_column!(df::DataFrame,
                                dv::AbstractVector,
-                               col_ind::ColumnIndex)
+                               col_ind::Union{Integer, Symbol})
```

Comment:
I think I had noted this before: wouldn't it be better to change `ColumnIndex` once for all?